### PR TITLE
Updating values directly from ETE

### PR DESCRIPTION
**Pull request by grdw**
Please consider that #492 needs to merged first.

### CODE REVIEW

**Comment by ChaelKruip** [comment]
The title and the content of this PR do not seem to belong together @grdw

**Comment by grdw** [comment]
The first two commits belong to PR #492

File: app/models/import/variable_om_costs_per_full_load_hour_attribute.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Import
+  VariableOmCostsPerFullLoadHourAttribute =
+    Attribute.new('fixed_om_costs_per_year', 'variable_operation_and_maintenance_costs_per_full_load_hour')
```
Comment (antw) [inline app/models/import/variable_om_costs_per_full_load_hour_attribute.rb]: I think the `local_name` argument ("fixed_om_costs_per_year") may be wrong. Was it meant to be "om_costs_per_full_load_hour"?

Comment (grdw) [inline app/models/import/variable_om_costs_per_full_load_hour_attribute.rb]: Yes. It should be. Moment please.

**Comment by antw** [comment]
Had to merge manually due to a conflict, but it looks good and works nicely. Thanks! :+1:

**Pull request closed without merging**